tests/appsec/iast/sink/test_sql_injection.py

```diff
@@ -18,10 +18,14 @@ class TestSqlInjection(BaseSinkTest):
     data = {"username": "shaquille_oatmeal", "password": "123456"}
     location_map = {
         "java": "com.datadoghq.system_tests.iast.utils.SqlExamples",
-        "nodejs": "iast/index.js",
+        "nodejs": {"express4": "iast/index.js", "express4-typescript": "iast.ts"},
         "python": {"flask-poc": "app.py", "django-poc": "app/urls.py"},
     }
 
+    @bug(weblog_variant="express4-typescript", reason="Incorrect vulnerability location")
```

Comment:
it was a known bug, or you discovered it during this PR?

Reply:
It has been discovered during this PR.